subprocess.h

```diff
@@ -44,6 +44,11 @@
 #include <stdio.h>
 #include <string.h>
 
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
No-way am I including `windows.h` in this library, it bloats compile time by 10x.

Reply:
I thought so.

I'll try to find a fix then. Getting error 87 on CreateProcess, which means the function signature is wrong.